cluster-autoscaler/go.mod

```diff
@@ -10,11 +10,10 @@ require (
 	github.com/Azure/go-autorest/autorest/date v0.3.0
 	github.com/Azure/go-autorest/autorest/to v0.2.0
 	github.com/aws/aws-sdk-go v1.35.24
-	github.com/codegangsta/negroni v1.0.0 // indirect
 	github.com/digitalocean/godo v1.27.0
 	github.com/ghodss/yaml v1.0.0
 	github.com/golang/mock v1.4.4
-	github.com/gorilla/context v1.1.1 // indirect
+	github.com/google/uuid v1.2.0
```

Comment:
Is it possible to use 1.1.2 ? Kubernetes/kubernetes uses 1.1.2 and we would rather like to stay consistent.

Reply:
Done